miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultLibraryDilutionService.java

```diff
@@ -204,7 +204,7 @@ private void loadChildEntities(LibraryDilution dilution) throws IOException {
     if (dilution.getLibrary() != null) {
       dilution.setLibrary(libraryDao.get(dilution.getLibrary().getId()));
     }
-    if (dilution.getTargetedSequencing() != null) {
+    if (dilution.getTargetedSequencing() != null && dilution.getTargetedSequencing().getId() != null) {
```

Comment:
Is there a reason the targeted sequencing is set and its id isn't? Seems like something an exception should be thrown for